resources/js/components/bankForms/CreateAccountPage.js

```diff
@@ -0,0 +1,33 @@
+import React,{useCallback} from 'react';
+import { useSelector } from 'react-redux';
+import { Redirect } from 'react-router';
+import JsonForm from '../../../../json-form-react/JsonForm';
+
+import { accountCreation } from '../../services/authService';
+
+const createAccountJson = require('../../../../forms/createAccount.json');
+
+function CreateAccountPage() {
+
+  const token = useSelector((state) => state.auth.token);
+
+  const onSubmit = useCallback((values) => {
+    accountCreation(token,values.fullname, values.address);
+  }, []);
+
+
+  // if logged out
+  if (!token) {
+    return <Redirect to="/logout"/>;
+  }
+  console.log('gets into the create account js');
```

Comment:
@raghuhvarman Just a note, the log you put here will print out every time something updates within this component (state or props). Also should remove logs when releasing code.

Reply:
Fixed